cmd/thor/node/node.go

```diff
@@ -153,6 +161,7 @@ func (n *Node) handleBlockStream(ctx context.Context, stream <-chan *block.Block
 	var blk *block.Block
 	for blk = range stream {
 		if blk == nil {
+			logger.Debug("received nil from stream")
```

Comment:
I wonder if this should be an error ? I mean it should never happen right ? but if it does it might indicative that somethings wrong. 
what do you think ?